app/assets/javascripts/results/result_assets.js

```diff
@@ -35,18 +35,18 @@
     }
 
     function applyEditResultAssetCallback() {
-      $('.edit-result-asset').on('ajax:success', function(e, data) {
+      $('.edit-result-asset').off().on('ajax:success', function(e, data) {
```

Comment:
It is quite dangerous to use `.off()` it can remove all events. Check this:
```
var foo = function() {
  // Code to handle some kind of event
};
 
// ... Now foo will be called when paragraphs are clicked ...
$( "body" ).on( "click", "p", foo );
 
// ... Foo will no longer be called.
$( "body" ).off( "click", "p", foo );
```